notes/stories/film/film-scifi-1990s.st.txt

```diff
@@ -11674,6 +11674,47 @@ speculative nanotechnology [Squid boasted of having "tweaked up" SETH's human bo
 chemical weapons [The military made the mistake of storing chemical weapons in the same facility that house the gone rogue sentient computer SETH.]
 
 
+movie: The Astronauts Wife (1999)
+=================================
+
+:: Title
+The Astronaut's Wife
+
+:: Description
+A woman's astronaut husband returns from space strangely altered.
+
+:: Date
+1999-08-27
+
+:: References
+https://en.wikipedia.org/wiki/The_Astronaut%27s_Wife
+
+:: Ratings
+2 <paul>
+
+:: Genre
+sci-fi
+
+:: Major Themes
+husband and wife [The main storyline follows Jillian as she deals with her astronaut husband, Spencer, acts increasing deranged after returning from a mission to repair a satellite. Alex and Natalie Streck.]
```

Comment:
Weird comment - typo plus a sentence fragment?

Reply:
Well, that is admittedly a train wreck of a motivating comment. It's been updated.